setup.py

```diff
@@ -22,8 +33,7 @@
     packages=find_packages(),
     long_description=open(join(prj_dir, 'README.rst')).read(),
     entry_points={
-        'console_scripts': ["ir-provision = cli.provision:main",
-                            "ir-install = cli.install:main"]
+        'console_scripts': generate_entry_scripts()
     },
     install_requires=reqs,
     author='Yair Fried',
```

Comment:
Since this is a community module, I think it will be o.k to change it to "rhos-qe" :)